.circleci/config.yml

```diff
@@ -481,7 +481,7 @@ workflows:
     when:
       and:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
-        - equal: [ "load-shedder-integration-tests", << pipeline.schedule.name >> ]
+        - equal: [ "load-shedder-integration-tests-v5", << pipeline.schedule.name >> ]
```

Comment:
Looks like we can have multiple triggers in CircleCI with the same trigger name... I changed this so we can run these and have v6 run their own